frontend/package.json

```diff
@@ -45,30 +44,10 @@
     "react-test-renderer": "^16.8.6",
     "whatwg-url": "^7.0.0"
   },
-  "proxy": {
-    "/query": {
-      "target": "http://localhost:8080"
-    },
-    "/export": {
-      "target": "http://localhost:8080"
-    },
-    "/schema": {
-      "target": "http://localhost:8080"
-    },
-    "/detect-lang": {
-      "target": "http://localhost:8080"
-    },
-    "/parse": {
-      "target": "http://localhost:8080"
-    },
-    "/get-languages": {
-      "target": "http://localhost:8080"
-    },
-    "/filter": {
-      "target": "http://localhost:8080"
-    },
-    "/version": {
-      "target": "http://localhost:8080"
-    }
-  }
+  "browserslist": [
```

Comment:
why do we need to define it here? I don't see anything about browserslist in the migration guide. Does `create-react-app` have some any defaults?

Reply:
The first time you run `yarn start` it complains that `browserslist` is not defined, and offers to fill it with the defaults you see here. I just left the defaults.